app/posts/register-trainee-teachers/2020-09-30-beta-prototype.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Beta prototype v1
+description:
+date: 2020-10-12
+related:
+  items:
+  - text: Current beta prototype (Heroku)
+    href: https://register-prototype.herokuapp.com/
+  - text: Github repo for prototype
+    href: https://github.com/DFE-Digital/register-trainee-teachers-prototype
+  - text: Alpha prototype
+    href: /register-trainee-teachers/alpha-prototype/
+---
+
+## Building a new prototype from scratch
```

Comment:
Not sure you need this heading – perhaps it could be the summary for this post instead?

Reply:
👍